Wrappers/Python/cil/framework/labels.py

```diff
@@ -121,12 +121,18 @@ class ImageDimension(_DimensionBase, StrEnum):
     """
     Available dimension labels for image data.
 
+    Attributes
+    ----------
+    CHANNEL
+    VERTICAL
+    HORIZONTAL_X
+    HORIZONTAL_Y
+
```

Comment:
this isn't useful IMO

Reply:
We made the labels render in the documentation so maybe useful to be able to tell what the attributes are from there?

Reply:
ok but shouldn't the attribs render automatically without hardcoding them in a docstring?

Reply:
Ooh ok, I think I figured out how to do that. Still looks slightly weird with the spacing.